pipeline/transformer/transformer_marc_common/src/test/scala/weco/pipeline/transformer/marc_common/transformers/MarcOrganisationTest.scala

```diff
@@ -64,7 +64,7 @@ class MarcOrganisationTest
           )
       }
 
-      val nonLabelSubfields = Seq("n", "0", "u") map {
+      val nonLabelSubfields = Seq("n", "2", "u") map {
```

Comment:
Using 0 here caused its value to be interpreted as an identifier, clouding the actual point of this test.